packages/language-support/src/parserWrapper.ts

```diff
@@ -174,8 +176,14 @@ export function createParsingResult(query: string): ParsingResult {
       const labelsCollector = new LabelAndRelTypesCollector();
       const variableFinder = new VariableCollector();
       const methodsFinder = new MethodsCollector(tokens);
+      const preparserFinder = new PreparserCollector();
```

Comment:
Every collector is in charge of extracting one type of structure here. I'd say say `cypherVersionCollector` here is more consistent with how we've named the rest